tests/texture_test_slow.toit

```diff
@@ -129,6 +129,7 @@ barcode_factory version code x y orientation:
       return true_color.BarCodeEan13 code x y transform
   unreachable
 
+/*
```

Comment:
Add a TODO with a link to an issue?
(unless you plan to fix this within the next days).